src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page">
+    <div class="card" data-qa="card">
+
+      <img
+        src="images\imac.jpeg"
+        alt="product-image"
+        class="card__img"
+      >
+
+      <div class="main-text">
+        <p class="main-text__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+        <p class="main-text__number">
+          Код товара: 195434
+        </p>
+      </div>
+
+      <div class="card__reviews">
+
+        <div class="stars">
+          <div class="stars-item"></div>
+          <div class="stars-item"></div>
+          <div class="stars-item"></div>
+          <div class="stars-item"></div>
+          <div class="stars-inactive"></div>
+        </div>
```

Comment:
`4. [TASK] - Make sure to reuse part of [Stars task](https://github.com/mate-academy/layout_stars) here, and keep it operational in case you change block modifier.` 

There should't be inactive stars class, use modifier, the same as in `stars` task.

always check your task according to task [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)